run_service_ssh.sh

```diff
@@ -1,14 +1,24 @@
 #!/bin/bash
 
+# This script first runs the ggp ssh init command to get the needed 
+# credentials to connect via ssh. Then it connects to the instance 
+# with portforwarding and starts OrbitService. To also deploy OrbitService,
+# invoke with a the argument -deploy and a path to the OrbitService executable
+# e.g. run_service_ssh.ps1 -deploy build/bin/OrbitService
```

Comment:
Update for Linux script (this says .ps1).

Reply:
done